src/components/PriceDisplay/PriceDisplay.js

```diff
@@ -27,7 +27,7 @@ const Top = styled.div`
 
 const Bottom = styled.div`
 	display: flex;
-	justify-content: flex-end;
+	justify-content: end;
```

Comment:
`end` is not a valid property here, either remove or set to a proper value.
https://www.w3schools.com/csSref/css3_pr_justify-content.asp

Reply:
9f32b1a
0786974
f7b6c89